app/docs/0.13.x/proxy.md

```diff
@@ -1074,6 +1074,26 @@ standard HTTP proxy.
 
 [Back to TOC](#table-of-contents)
 
+### WebSocket and TLS
```

Comment:
This needs to be added to the ToC at the beginning of this document.

Also, it would be nice to add the same paragraph for the 0.11 and 0.12 versions of this document at least.